guides/common/modules/proc_installing-webhooks-shellhooks-plugin.adoc

```diff
@@ -0,0 +1,28 @@
+[id="installing-webhooks-shellhooks-plugin_{context}"]
```

Comment:
Why does this anchor contain `shellhooks`? I thought webhooks are deprecating shellhooks.